<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alterarSenha</class>
 <widget class="QDialog" name="alterarSenha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>314</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>392</width>
    <height>314</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>392</width>
    <height>314</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="aviso7_label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="username_label">
         <property name="text">
          <string>username:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="usernamelineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="senhaAtuallabel">
           <property name="text">
            <string>senha atual:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="senhaAtuallineEdit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="avisolabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="novaSenhalabel">
           <property name="text">
            <string>nova senha:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="novaSenhalineEdit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="avisolabel_2">
           <property name="text">
            <string>aqui</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="confirmarlabel">
           <property name="text">
            <string>confirmar nova senha:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="confirmarlineEdit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="saveBtn">
           <property name="text">
            <string>Salvar</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>148</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cancelBtn">
           <property name="text">
            <string>Cancelar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
